core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -332,6 +330,29 @@ public HandleCallbackDecorator getHandleCallbackDecorator() {
         return this.handleCallbackDecorator.get();
     }
 
+    /**
+     * Returns the internal {@link HandleScope} object. The Jdbi instance uses this to provide handles in a given scope.
+     * The default scope is <i>per-thread</i>, so every thread manages its own handle.
+     * <br>
+     * <b>This is an internal method and not part of the public API!</b>
+     * @return A {@link HandleScope} object
+     * @param <T> type parameter
+     */
+    @Alpha
```

Comment:
Non-public API is never supported or stable, so `@Alpha` doesn't make too much sense here IMO

Reply:
removed it. Also put handlescope into the internal package, so it is not part of the public API footprint